.vscode/launch.json

```diff
@@ -1,6 +1,13 @@
 {
     "version": "0.2.0",
     "configurations": [
+        {
```

Comment:
unrelated to current PR, this allows vscode users to run python files in the debugger